pkg/token/options.go

```diff
@@ -59,6 +61,7 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&o.LoginMethod, "login", "l", o.LoginMethod, fmt.Sprintf("Login method. Supported methods: %s. It may be specified in %s environment variable", GetSupportedLogins(), envLoginMethod))
 	fs.StringVar(&o.ClientID, "client-id", o.ClientID, fmt.Sprintf("AAD client application ID. It may be specified in %s environment variable", envServicePrincipalClientID))
 	fs.StringVar(&o.ClientSecret, "client-secret", o.ClientSecret, fmt.Sprintf("AAD client application secret. Used in spn login. It may be specified in %s environment variable", envServicePrincipalClientSecret))
+	fs.StringVar(&o.ClientCert, "client-cert", o.ClientCert, fmt.Sprintf("AAD client application cert. Used in spn login. It may be specified in %s environment variable", envServicePrincipalClientCert))
```

Comment:
```suggestion
	fs.StringVar(&o.ClientCert, "client-certificate", o.ClientCert, fmt.Sprintf("AAD client application cert. Used in spn login. It may be specified in %s environment variable", envServicePrincipalClientCert))
```

Reply:
please specify whether it should be pfx or pem (or both?)